### DISCUSSION TITLE
Invalid type for column-name error

### DISCUSSION THREAD

**Discussion by aryala7** [spatie/laravel-ciphersweet#32] [General]
after the last update if value of the desired column is null, the command throws an invalid type error:
this is my method in user model:
`    public static function configureCipherSweet(EncryptedRow $encryptedRow): void
    {
        $encryptedRow
            ->addField('email')
            ->addField('cellphone');
    }`
    in the database cellphone is  NULL

![image](https://user-images.githubusercontent.com/56627259/230714963-2f7e1d29-deae-4317-b1f0-7c91a1ec8435.png)

**Comment by freekmurze** [comment]
@riasvdv could you have a look?

**Comment by AvenueShops** [comment]
I'm having the same issue. Brand new laravel/ciphersweet package install (v1.2, laravel 9) and running encrypt for the first time. It looks like paragonie/ciphersweet added the scalar check in [v4.3.0](https://github.com/paragonie/ciphersweet/blob/v4.3.0/src/EncryptedRow.php). I don't see anyone reporting issues on paragonie/ciphersweet discussions though. I was able to get past that issue by specifying a previous version of paragonie/ciphersweet in my composer.json file, like:
`"paragonie/ciphersweet": "4.2.*",`

**Comment by stevenmaguire** [comment]
I'd like to confirm the same observations as @AvenueShops above. I am also encountering the same error originally reported here. I was also able to work around the issue by specifying the older upstream package version.

**Comment by stevenmaguire** [comment]
I've created an issue on the upstream package.

https://github.com/paragonie/ciphersweet/issues/93

**Reply by aryala7** [reply]
@stevenmaguire it has been fixed in the latest release. try using optional column types 
` ->addOptionalTextField('email')`

**Reply by jdecode** [reply]
This worked for me!
